src/event-notion/NotionCalEvent.ts

```diff
@@ -118,12 +118,18 @@ const filterOrgsResponse = (response: HostFormResponse): StudentOrg[] => {
  * @returns The Interval in which a NotionCalEvent takes place.
  */
 const getEventInterval = (response: HostFormResponse): Interval => {
-  return Interval.fromDateTimes(
+  let interval = Interval.fromDateTimes(
     // Convert from the provided host form responses to the DateTime objects.
     // The format string below SEEMS to work for most events, but I MAY be wrong.
     DateTime.fromFormat(`${response['Preferred date']} ${response['Preferred start time']}`, 'M/d/yyyy h:mm:ss a'),
     DateTime.fromFormat(`${response['Preferred date']} ${response['Preferred end time']}`, 'M/d/yyyy h:mm:ss a'),
   );
+
+  if (interval.toDuration().minutes >= 0) {
+    return interval;
+  }
```

Comment:
nit: I think the error check here is a little unintuitive here: Luxon has a Interval property called `.isValid()` that returns if the interval is in the right order.

even tinier nit: I also think it makes more sense to throw the error inside the if block, and return the value after we go past the if statement